stable/doc_hardware.md

```diff
@@ -1 +1,107 @@
+
+![](./doc_hardware/block_diagram.png)
+
+The device generate a high voltage pulse to excite the transducer (*via* or not an impedance matching layer). The rest of the device is protected from the high voltage pulse by a T/R switch. The signal is amplified by a TGC (variable amplification driven by the controller), filtered (pass-band filter) and converted to digital signal. The signal is the send *via* wifi to a terminal for display.
+
+For making a 2D image, the transducer must make a sweeping movement. The transducer is placed on the choosen line by the controller, acquisition is made and send to the terminal, then the transducer is moved to the next line. The lines are angulary equally reparted on the sector of measurement.
+
+## Device informations
+
+### Acoustic
+
+We use a commercial transducer from imasonic with a central frequency of 3.27 MHz. Technical informations about the transducer (temporal and frequential pulse respectively) can be found in the following images
+
+![](./doc_hardware/signalt.png "Temporal pulse of the transducer") 
+
+![](./doc_hardware/signalf.png "Frequential pulse of the transducer")
+
+Transducer characteristics:
+
+* Pulse duration at -6 dB: 382 ns (precision of 0.56 mm)
+* Acoustic pass-band: 2.13 MHz, from 2.21 to 4.33 MHz
+* Focal depth: 120 mm
+* Excitation time: around half the period = 142 ns
+* Excitation voltage: < -150 V
+
+### Electronic (analogic)
+
+Actually we still use two evaluation kits:
+
+* Pulser: MAX4940EVKIT
+* TGC: AD8331-EVAL
+
+and three commercial boards:
+
+* Stepper driver: A4988
+* Arduino: arduino nano
+* Acquisition: RedPitaya
+
+We use an arduino nano because the GPIO of the RedPitaya are two slow to drive the pulser (minimum 500 ns), with the arduino (16 MHz clock) the command can be a multiple of 62.5 ns.
+
+The output of the high voltage circuit is around -97 V.
+
+T/R switch input must been in range +/- 100 V, output is between +/- 2 V.
+
+Pass-band filter is centered around 3.5 MHz.
+
+TGC caracteristic:
+
+* Amplification command: between 0 and 1 V
+* Amplification on LO gain mode: from -4.5 to +43.5 dB
+* Amplification on HI gain mode: from +7.5 to +55.5 dB
+* Output: between -2.5 and +2.5 V
+
+
+### Mechanic
+
+For making a sector image, we must transmit a sweeping movement to the transducer. This is done with a stepper motor. With the stepper driver we can easily change the direction of the steps. At start the motor turn and fix to a mechanical stop (so we now it's position) and then we make the sweeping movement.
+
+### Acquisition
+
+Acquisition is made with a RedPitaya which caracteristics are:
+
+* Sampling rate: 125 or 125/8 Msps
+* Precision: 14 bits
+* Voltage range: +/- 1 V or +/- 20 V
+* Buffer maximum size: 16384 points
+
+Considering that the acoustic waves makes back and forth between the transducer and the medium, depth of measurement is given by d=v*t/2. So maximum depth of measurement at 125 Msps is **97 mm** and at 125/8 Msps it is **775 mm**. 
```

Comment:
The discussion about maximal depth should rather be in characteristics.md.
Besides, I don't understand at all the explanations given here. What's the link between sampling rate and depth of detection ?! Why do the value of the sampling rate influences the maximal depth ?!